foodhub/accounts/models.py

```diff
@@ -0,0 +1,145 @@
+from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.contrib.auth.base_user import BaseUserManager
+
+# Create your models here.
+
+
+class UserManager(BaseUserManager):
+    def create_user(self, email, password, **extra_fields):
+        if not email:
+            raise ValueError('The email must be set')
+
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save()
+        return user
+
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault('is_staff', True)
+        extra_fields.setdefault('is_superuser', True)
+        extra_fields.setdefault('is_active', True)
+
+        if extra_fields.get('is_staff') is not True:
+            raise ValueError('Superuser must have is_staff=True.')
+        if extra_fields.get('is_superuser') is not True:
+            raise ValueError('Superuser must have is_superuser=True.')
+        return self.create_user(email, password, **extra_fields)
+
+
+class User(AbstractUser):
+    username = None
+    email = models.EmailField(
+        max_length=150, blank=True, null=True, unique=True)
+    USERNAME_FIELD = 'email'
+    is_active = models.BooleanField(default=True)
+    REQUIRED_FIELDS = []
+
+    def __str__(self):
+        return self.email
+
+    objects = UserManager()
+
+    def save(self, *args, **kwargs):
+        super(User, self).save(*args, **kwargs)
+
+    class Meta:
+        db_table = 'users'
+
+# Field Choices
+
+class OrderStatus(models.TextChoices):
+    ACTIVE = 'ACT', 'Active'
+    CANCELLED = 'CAN', 'Cancelled'
+    FINISHED = 'FIN', 'Finished'
```

Comment:
Just a sugg, instead of Active it can be Placed, similarly instead of finished it can be Completed.

Reply:
As per requirement i have updated my models.py file